manual-tests/automated-test.py

```diff
@@ -0,0 +1,342 @@
+
+p = "zenoh-zf-plugin-01.json".__len__()
+if zf_conf.zf_plugin_path[-p:] != "zenoh-zf-plugin-01.json":
+    print("[Info] Zenoh-Flow as Zenoh plugin")
+    if os.path.exists(os.path.join(manual_tests_path, "zenoh-zf-plugin.json")) is False:
+        print(zenoh_zf_plugin_path)
+        shutil.copy(zenoh_zf_plugin_path, manual_tests_path)
+    add_lib_folder = ""
+
+    zf_conf.zf_plugin_path = os.path.join(manual_tests_path, "zenoh-zf-plugin.json")
+    with open(zenoh_zf_plugin_path) as json_file:
+        json_string = json_file.read()
+        if json_string.find(dir_zenoh_zf_plugin) == -1:
+            start_row = json_string.find('"plugins_search_dirs":[')
+            index = (start_row - 1) + '"plugins_search_dirs": ['.__len__()
+            add_lib_folder = (
+                json_string[:index]
+                + '"'
+                + dir_zenoh_zf_plugin
+                + '",'
+                + json_string[index:]
+            )
+
+    if add_lib_folder != "":
+        with open(zf_conf.zf_plugin_path, "w", encoding="utf8") as json_file:
+            json_file.write(add_lib_folder)
+
+print("[Info] Commands:")
+if (
+    zf_conf.zenohd_path != home_path + "/.local/bin/zenohd"
+    or zf_conf.zfctl_path != home_path + "/.config/zenoh-flow/zfctl.json"
+):
+    zf_command["zenohd_cmd"] = zf_conf.zenohd_path + " -c " + zf_conf.zf_plugin_path
+    print(f'Lauch Zenoh with Zenoh-Flow plugin: \n {zf_command["zenohd_cmd"]}')
+    zf_command["runtimes_cmd"] = zf_conf.zfctl_path + " list runtimes"
+    print(f'show the list of runtimes: \n {zf_command["runtimes_cmd"]}')
+    zf_command["dataflow_yaml_cmd"] = zf_conf.zfctl_path + " launch " + data_flow_path
+    print(f'Lauch Zenoh-Flow configuration: \n {zf_command["dataflow_yaml_cmd"]}')
+    zf_command["instances_cmd"] = zf_conf.zfctl_path + " list instances"
+    print(f'show list of Zenoh-Flow instances: \n {zf_command["instances_cmd"]}')
+else:
+    zf_command["zenohd_cmd"] = zf_conf.zenohd_path + " -c " + zf_conf.zf_plugin_path
+    print(f'Lauch Zenoh with Zenoh-Flow plugin: \n {zf_command["zenohd_cmd"]}')
+    zf_command["runtimes_cmd"] = (
+        "ZFCTL_CFG="
+        + zf_conf.zfctl_path
+        + " "
+        + home_path
+        + "/.local/bin/zfctl"
+        + " list runtimes"
+    )
+    print(f'show the list of runtimes: \n {zf_command["runtimes_cmd"]}')
+    zf_command["dataflow_yaml_cmd"] = (
+        "ZFCTL_CFG="
+        + zf_conf.zfctl_path
+        + " "
+        + home_path
+        + "/.local/bin/zfctl"
+        + " launch "
+        + data_flow_path
+    )
+    print(f'Lauch Zenoh-Flow configuration: \n {zf_command["dataflow_yaml_cmd"]}')
+    zf_command["instances_cmd"] = (
+        "ZFCTL_CFG="
+        + zf_conf.zfctl_path
+        + " "
+        + home_path
+        + "/.local/bin/zfctl"
+        + " list instances"
+    )
+    print(f'show list of Zenoh-Flow instances: \n {zf_command["instances_cmd"]}')
+
+print("[Info] Looking for Zenohd process...")
```

Comment:
Given that you are not just looking for `zenohd` processes, I would rather write:
```suggestion
print("[Info] Killing extra `zenohd` processes...")
```